applets/brightness/brightness-applet.c

```diff
@@ -484,25 +492,21 @@ gpm_applet_key_press_cb (GtkWidget *popup, GdkEventKey *event, GpmBrightnessAppl
 		}
 		break;
 	case GDK_KEY_Page_Up:
-		for (i = 0;i < 10;i++) {
-			gpm_applet_plus_cb (NULL, applet);
-		}
+		gpm_applet_plus_cb (NULL, applet, 10);
 		return TRUE;
 		break;
 	case GDK_KEY_Left:
 	case GDK_KEY_Up:
-		gpm_applet_plus_cb (NULL, applet);
+		gpm_applet_plus_cb (NULL, applet, 5);
```

Comment:
Shouldn't it be 1 here instead of 5?

Reply:
Yes it is, but what's the point of adding brightness value one at a time? if i want to bring the brightness from 1 to 100 i should click 99 times!  all other desktop environments do it by 5 or 10 values at a time.

Reply:
It's a key press handler, and keys can be held down for some time.